composites/Plugin/Shared/components/Collapsible.js

```diff
@@ -67,16 +77,21 @@ const StyledSubTitle = styled.span`
 /**
  * Wraps a component in a heading element with a defined heading level.
  *
- * @param {ReactElement} Component    The component to wrap.
- * @param {int}          headingLevel The heading level.
+ * @param {ReactElement} Component        The component to wrap.
+ * @param {object}       props            The heading props.
+ * @param {boolean}      props.level      The heading level.
+ * @param {boolean}      props.fontSize   The heading font-size.
+ * @param {boolean}      props.fontWeight The heading font-weigth.
  *
  * @returns {Function} A function that will return the wrapped component with given properties.
  */
-export function wrapInHeading( Component, headingLevel ) {
-	const Heading = `h${ headingLevel }`;
+export function wrapInHeading( Component, props ) {
+	const Heading = `h${ props.level }`;
 	const StyledHeading = styled( Heading )`
-		margin: 0;
-		font-weight: normal;
+		margin: 0 !important;
+		padding: 0 !important;
+		font-size: ${ props.fontSize } !important;
```

Comment:
Is there a specific reason as to why `fontSize` and `fontWeight` aren't in the prop types list? Additionally, adding a default would allow you to leave out some stuff such at the `fontWeight: "normal"` on line 106.

Reply:
They are in the props type list but as sub-item: headingProps.
As this function is not part of the class, the props are not the same props, but can be given as object constructed from anywhere.
I wanted to keep them similar to the ones used in `headingProps` to maintain reference.

Reply:
We should always avoid to use `!important`.

Reply:
To expand on this, there's a better way to increase specificity when necessary, see 
https://github.com/Yoast/yoast-components/blob/dac4110ea3519488d0ea42e18bfbcc1e79a3b0a3/composites/AlgoliaSearch/SearchBar.js#L36

the reason why this approach is preferable is that `!important` can't be overridden other than using another `!important` while the second approach can be overridden in an easier way, if need be. Will open an issue for consideration.